helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -30,17 +30,11 @@ data:
       -Dshare.protocol={{ tpl (.Values.externalProtocol | default "http") $ }}
       -Dshare.host={{ tpl (.Values.externalHost | default (printf "%s-share" (include "content-services.shortname" .))) $ }}
       -Dshare.port={{ tpl (.Values.externalPort | default .Values.share.service.externalPort | toString) $ }}
-      {{- if and (not (index .Values "alfresco-search" "enabled")) (not (index .Values "alfresco-search-enterprise" "enabled")) (not (index .Values "alfresco-search" "external" "host")) }}
-      -Dindex.subsystem.name=none
-      {{- else if or (index .Values "alfresco-search" "enabled") (index .Values "alfresco-search" "external" "host") }}
-      -Dindex.subsystem.name=solr6
-      -Dsolr.base.url={{ template "alfresco-search.baseurl" . }}
-      -Dsolr.host={{ template "alfresco-search.host" . }}
-      -Dsolr.port={{ template "alfresco-search.port" . }}
-      -Dsolr.secureComms={{ .Values.global.tracking.auth | default "secret" }}
-      {{- else if index .Values "alfresco-search-enterprise" "enabled" }}
-      {{- include "repo.elasticsearch.config" . | nindent 6 }}
+      {{- $subsys := include "repository.indexSubsystem" . }}
+      {{- if ne $subsys "none" }}
+      {{- include "repository.indexConfig" . | indent 6 }}
       {{- end }}
+      -Dindex.subsystem.name={{ $subsys }}
```

Comment:
I was doing a similar refactor in my PR and it would require a manual merge, better avoid any cleanup outside the current chart being migrated until we finished with migrating charts

Reply:
I'm afraid I need this one as explained this morning